### PR TITLE
Add glob2 recipe

### PR DESCRIPTION
**Pull request by dfroger**

### CODE REVIEW

**Comment by dfroger** [comment]
Note: My goal is to package the [craftr](https://github.com/craftr-build/craftr) build system, this is one of its depenencies.

File: recipes/glob2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{% set version = "0.5" %}
+
+package:
+  name: glob2
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
```
Comment (scopatz) [inline recipes/glob2/meta.yaml]: The filename should have the package name in it

**Comment by scopatz** [comment]
Looks good, just the one minor issue.

**Comment by dfroger** [comment]
Just fixed the source filename issue. Thanks!

**Pull request merged**